src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -154,17 +154,29 @@ <h3>Active courses</h3>
   <h2 class="text-muted">
     <span class="fa fa-file-archive"></span> Archived Courses
   </h2>
-  <div class="table-responsive">
-    <table id="archived-courses-table" class="table table-striped table-bordered margin-0">
-      <thead>
-        <tr class="bg-info text-white">
-          <th>Course ID</th>
-          <th>Course Name</th>
-          <th>Creation Date</th>
-          <th class="text-center">Action(s)</th>
-        </tr>
-      </thead>
-      <tbody>
+  <div class="card top-padded">
+    <div class="card-header archive-course-header bg-info" (click)="isArchivedCourseExpanded = !isArchivedCourseExpanded">
```

Comment:
Might want to do something with `recycle-bin-header`as well, like giving it `cursor: pointer`? I don't think that class is doing anything currently.